pkg/operator/ceph/object/bucket/controller.go

```diff
@@ -53,6 +55,10 @@ type ReconcileBucket struct {
 // Add creates a new Ceph CSI Controller and adds it to the Manager. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager, context *clusterd.Context, opManagerContext context.Context, opConfig opcontroller.OperatorConfig) error {
+	if os.Getenv(object.DisableOBCEnvVar) == "true" {
+		logger.Info("skip running Object Bucket controller")
+		return nil
```

Comment:
Also, can you add a unit test for it?

Reply:
We don't have unit tests for the controllers since they are so high level and difficult to mock all the context. So for this simple change, I'd say unit tests aren't needed.